src/clusterfuzz/_internal/cron/external_testcase_reader.py

```diff
@@ -31,6 +32,15 @@
 ISSUETRACKER_WONTFIX_STATE = 'NOT_REPRODUCIBLE'
 
 
+def has_vrp_upload_permission(reporter):
+  """Checks whether the given reporter has permission to upload."""
+  storage_client = storage.Client()
+  bucket = storage_client.bucket('clusterfuzz-vrp-uploaders')
+  blob = bucket.blob('vrp-uploaders')
+  members = blob.download_as_string().decode('utf-8').splitlines()[0].split(',')
```

Comment:
Might be a good idea to have a YAML file but I'm OK with this.

Reply:
true!! ive added a comment for me to do!